charts/orc-crds/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright Copyright 2025 The ORC Authors.
+# SPDX-License-Identifier: APACHE-2.0
+
+apiVersion: v2
```

Comment:
Does that refer to the apiVersion of the CRDs? If that's the case, this should be `v1alpha1`

Reply:
Nope it's helm version ^^